docs_user/modules/proc_adopting-networker-services-to-the-data-plane.adoc

```diff
@@ -214,51 +196,55 @@ endif::[]
         - openvswitch3.1
 EOF
 ----
++
+<1> If TLS Everywhere is enabled, change `spec:tlsEnabled` to `true`.
+<2> Set to the same values that you used in your {rhos_prev_long} {rhos_prev_ver} deployment.
+<3> Set to `true` to run `ovn-controller` in gateway mode.
 
-. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the `Networker` nodes before adoption. This configuration is stored in the `external_ids`` column in the `Open_vSwitch` table in the Open vSwitch database:
+. Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as the settings that were set in the `Networker` nodes before adoption. This configuration is stored in the `external_ids`` column in the `Open_vSwitch` table in the Open vSwitch database:
```

Comment:
"Ensure that you use the same `ovn-controller` settings in the `OpenStackDataPlaneNodeSet` CR that you used in the `Networker` nodes before adoption."